index.html

```diff
@@ -844,9 +844,7 @@ <h3>DID URL Syntax</h3>
       </table>
 
       <p class="note" title="Semicolon character is reserved for future use">
-This specification reserves the semicolon (<code>;</code>) character for
-possible future use as a sub-delimiter for parameters as described in
-[[?MATRIX-URIS]].
+        Although the semicolon (<code>;</code>) character may be used according to the rules of the DID URL syntax, future versions of this specification may use it as a sub-delimiter for parameters as described in [[?MATRIX-URIS]]]. To avoid future conflicts, developers should restrain using it.
```

Comment:
```suggestion
        Although the semicolon (<code>;</code>) character may be used according to the rules of the DID URL syntax, future versions of this specification may use it as a sub-delimiter for parameters as described in [[?MATRIX-URIS]]]. To avoid future conflicts, developers should restrain from using it.
```

Reply:
Would be good to shift this left and line wrap like other similar sections.

Reply:
This does sound a bit too fuzzy.  Can the semicolon be used or not?  If people start using it, then won't it be extremely difficult for future spec updates to justify any change that would break those uses?